deepchecks/nlp/utils/text_properties.py

```diff
@@ -616,10 +651,22 @@ def _select_properties(
     if include_properties is not None and ignore_properties is not None:
         raise ValueError('Cannot use properties and ignore_properties parameters together.')
 
+    include_properties = [prop.title() for prop in include_properties] if include_properties else None
```

Comment:
We write it capitalized in all the places, because it's also a title for axes etc.
So thought I'll just change what the user gave us to this instead of the other way

Reply:
You were right about this